src/rdkafka_admin.c

```diff
@@ -5939,11 +5939,11 @@ rd_kafka_MemberDescription_new(const char *client_id,
                 return NULL;
```

Comment:
Defensive checks like this is typically not done internally in C apps/libs, and it makes things more complex since now MemberDescription_new() can return NULL, which needs to be checked by the caller.

Reply:
done